shared/checks/oval/installed_env_has_openssh-server_package.xml

```diff
@@ -0,0 +1,37 @@
+<def-group>
+  <definition class="inventory"
+  id="installed_env_has_openssh-server_package" version="1">
+    <metadata>
+      <title>Package openssh-server is installed</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <description>Checks if package openssh-server is installed.</description>
+      <reference ref_id="cpe:/a:openssh-server" source="CPE" />
+    </metadata>
+    <criteria>
+      {{% if product in ['opensuse', 'sle11', 'sle12'] %}}
+      <criterion comment="Package openssh is installed" test_ref="test_env_has_openssh_installed" />
+      {{% else %}}
+      <criterion comment="Package openssh-server is installed" test_ref="test_env_has_openssh-server_installed" />
+      {{% endif %}}
+    </criteria>
+  </definition>
+
+  <!-- opensuse, sle11, sle12 -->
+  <linux:rpminfo_test check="all" check_existence="at_least_one_exists" comment="system has package openssh installed" id="test_env_has_openssh_installed" version="1">
+    <linux:object object_ref="obj_env_has_openssh_installed" />
+  </linux:rpminfo_test>
+  <linux:rpminfo_object id="obj_env_has_openssh_installed" version="1">
+    <linux:name>openssh</linux:name>
+  </linux:rpminfo_object>
+
+  <!-- everything else -->
+  <linux:rpminfo_test check="all" check_existence="at_least_one_exists" comment="system has package openssh-server installed" id="test_env_has_openssh-server_installed" version="1">
+    <linux:object object_ref="obj_env_has_openssh-server_installed" />
+  </linux:rpminfo_test>
+  <linux:rpminfo_object id="obj_env_has_openssh-server_installed" version="1">
+    <linux:name>openssh-server</linux:name>
+  </linux:rpminfo_object>
+
```

Comment:
I think you're disabling the Debian and Ubuntu profiles by this. The  Debian and Ubuntu profiles also contain rules related to SSH but your platform CPE test depends solely on RPM which isn't used on these distributions.

Reply:
Yes, this should also support DEB packages.